hal/src/rtl872x/spi_hal.cpp

```diff
@@ -445,7 +470,6 @@ class Spi {
 
     uint8_t transfer(uint8_t data) {
         CHECK_TRUE(isEnabled(), 0);
-        CHECK_TRUE(isBusy() == false, 0);
```

Comment:
Removing this check allows the busy loop to spin below while a previous master transmission finishes, right? This kind of forces the system to block and prevent repeated `hal_spi_transfer()` calls from failing?

Reply:
Yes, it's consustent with Gen3. And this is the root cause why `SPI.transfer(byte)` loss data in some circumstances.